### PR TITLE
Bump up grpc to 1.50.0 to fix CVE-2022-3171

### PR DESCRIPTION
**Pull request by alanlvle**
- [ ] If this pull request closes/resolves/fixes an existing issue, replace the issue number. Closes #<issue number>.
- [ ] Update the [`CHANGES` log](https://github.com/apache/skywalking-java/blob/main/CHANGES.md).

### CODE REVIEW

**Comment by wu-sheng** [comment]
You should update change log and license to adopt this version change.

**Comment by alanlvle** [comment]
Which files are updated? need help

**Comment by wu-sheng** [comment]
The version of the binary license, https://github.com/apache/skywalking-java/blob/main/dist-material/LICENSE#L220

File: pom.xml [hunk 1]
```diff
@@ -87,7 +87,7 @@
 
         <!-- core lib dependency -->
         <bytebuddy.version>1.12.13</bytebuddy.version>
-        <grpc.version>1.44.0</grpc.version>
+        <grpc.version>1.50.0</grpc.version>
```
Comment (lujiajing1126) [inline pom.xml:90]: `protobuf` should also be updated to be aligned with `grpc`

**Review by lujiajing1126 — CHANGES_REQUESTED** [review]

File: pom.xml (hunk 1 above)
Comment (alanlvle) [reply pom.xml:90]: grpc have fix protobuf

Comment (wu-sheng) [reply pom.xml:90]: If it does, you should consider bumping up gRPC as well.

Comment (alanlvle) [reply pom.xml:90]: yes,I find gRPC vesion config use same configuration。 All update to 1.50.0

Comment (wu-sheng) [reply pom.xml:90]: `com.google.protobuf.protoc.version` and `protoc-gen-grpc-java.plugin.version` should be updated as well.

And please check the netty version grpc-java bundling, which our 4.1.79.Final is suitable or lower.

Comment (alanlvle) [reply pom.xml:90]: update it. check the netty version is ok.

File: CHANGES.md [hunk 2]
```diff
@@ -14,6 +14,7 @@ Release Notes.
 * Fix `Apache ShenYu` plugin traceId empty string value. 
 * Add plugin to support [brpc-java-3.x](https://github.com/baidu/starlight/tree/brpc-java-v3)
 * Update `compose-start-script.template` to make compatible with new version docker compose
+* Bump up grpc-protobuf to 1.50.0 to fix CVE-2022-3171
```
Comment (wu-sheng) [inline CHANGES.md:17]:
```suggestion
* Bump up grpc to 1.50.0 to fix CVE-2022-3171
```

**Review by lujiajing1126 — APPROVED** [review]
LGTM. Thanks!

**Pull request merged**